packages/common/src/enum/organizations.ts

```diff
@@ -24,3 +24,16 @@ export const EmailDomains = {
   'northernhealth.ca': Organizations.NORTHERN_HEALTH,
   'vch.ca': Organizations.VANCOUVER_COASTAL_HEALTH,
 };
+
+export const EmailDomainByAcronym = {
```

Comment:
Constants related to organization could be refactored but no idea at the moment.

Reply:
agreed